core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -100,6 +102,16 @@ class MultiCloudWorkspaceService(userInfo: UserInfo,
       _ <- traceWithParent("enableLeoInWSM", parentSpan)(_ =>
         Future(workspaceManagerDAO.enableApplication(workspaceId, wsmConfig.leonardoWsmApplicationId, userInfo.accessToken))
       )
+      _ = logger.info(s"Creating Azure relay in WSM [workspaceId = ${workspaceId}]")
+      azureRelayCreateResult <- traceWithParent("createAzureRelayInWSM", parentSpan)(_ =>
+        Future(workspaceManagerDAO.createAzureRelay(workspaceId, workspaceRequest.region, userInfo.accessToken))
+      )
+      relayJobControlId = azureRelayCreateResult.getJobReport.getId
+      _ = logger.info(s"Polling on Azure relay in WSM [workspaceId = ${workspaceId}, jobControlId = ${relayJobControlId}]")
+      _ <- traceWithParent("pollGetAzureRelayCreationStatusInWSM", parentSpan)(_ =>
+        pollWMCreation(workspaceId, relayJobControlId, userInfo.accessToken, 5 seconds,
```

Comment:
nitpick: any reason this polls every 5 seconds vs the 2 seconds during cloud context creation?

Reply:
I think our comments crossed. Cloud context creation is pretty much instant, but this the relay takes about 45 seconds to create. I didn't see any reason to be polling every 2 seconds because it adds so much noise to the logs, but can change it to be consistent if we want.